attest/ake/src/initiator.rs

```diff
@@ -73,10 +61,10 @@ where
 {
     type Error = Error;
 
-    fn try_next(
+    fn try_next<R: CryptoRng + RngCore>(
         self,
-        csprng: &mut (impl CryptoRng + RngCore),
-        _input: ClientInitiate<KexAlgo, Cipher, DigestType>,
+        csprng: &mut R,
+        mut _input: ClientInitiate<KexAlgo, Cipher, DigestType>,
```

Comment:
Why does an used parameter needs to be `mut`?

Reply:
At one point in development I tried to make that `try_next()` param be `mut`, but it's not actually possible to have a mutable param in a trait (I think this is rustc being too clever by half, tho), and just forgot to strip this.